lms/templates/student_account/form_field.underscore

```diff
@@ -40,7 +40,8 @@
            type="<%= type %>"
            name="<%= name %>"
            class="input-block <% if ( type === 'checkbox' ) { %>checkbox<% } %>"
-           aria-describedby="<%= form %>-<%= name %>-desc"
+           <% if ( type !== 'password' ) { %>
+           aria-describedby="<%= form %>-<%= name %>-desc"<% } %>
```

Comment:
The code formatting/indentation is a little confusing here. Maybe make this 3 lines?